deploy-manage/security.md

```diff
@@ -75,64 +74,186 @@ $$$maintaining-audit-trail$$$
+* Use the list of [Elastic Cloud static IPs](./security/elastic-cloud-static-ips.md) to allow or restrict communications in your infrastructure.
 
-- [**Elastic Cloud Hosted and Serverless**](security/secure-your-elastic-cloud-organization.md)
-- [**Elastic Cloud Enterprise**](security/secure-your-elastic-cloud-enterprise-installation.md)
-- [**Elastic Cloud on Kubernetes**](security/secure-your-eck-installation.md)
+::::{note}
+While serverless projects are fully managed and secured by Elastic, additional security settings are available for you to configure individually for your {{ech}} deployments.
+::::
 
-:::{note}
-There is no orchestration layer for self-managed deployments because you directly control the host environment. Refer to [](security/manually-configure-security-in-self-managed-cluster.md) to learn more about securing self-managed installations.
-:::
+Refer to [Elastic Cloud security](https://www.elastic.co/cloud/security) for more details about Elastic security and privacy programs.
+
+## Cluster or deployment security features
+
+You can configure the following aspects of your Elastic implementation to maintain and enhance security:
+
+### Manage TLS certificates
+```yaml {applies_to}
+deployment:
+  ece: all
+  eck: all
+  self: all
+```
+
+TLS certificates apply security controls to network communications. TLS is the modern name for what used to be called Secure Sockets Layer (SSL).
+
+TLS certificates are used in two places:
+* **The HTTP layer**: Used for communication between your cluster or deployment and the internet.
+* **The transport layer**: Used mainly for inter-node communications, and in certain cases for cluster to cluster communication.
+
+The way that TLS certificates are managed depends on your deployment type:
+
+* In self-managed {{es}} clusters, you [manage both of these certificates yourself](./security/secure-cluster-communications.md). You can also [Configure Kibana and Elasticsearch to use mutual TLS](./security/secure-http-communications.md#elasticsearch-mutual-tls).
+* In {{ece}}, you can use one or more [proxy certificates](./security/secure-your-elastic-cloud-enterprise-installation/manage-security-certificates.md) to secure the HTTP layer. These certificates are managed at the ECE installation level. Transport-level encryption is managed by ECE and certificates can’t be changed.
+* In {{eck}}, you can [manage certificates for the HTTP layer](./security/secure-http-communications.md#k8s-custom-http-certificate). Certificates for the transport layer are managed by ECK and can’t be changed. However, you can set your own certificate authority, customize certificate contents, and provide your own certificate generation tools using [transport settings](./security/k8s-transport-settings.md).
+
+::::{tip}
+Elastic Cloud manages TLS certificates for you.
+::::
+
+#### Enable cipher suites for stronger encryption
+
+TBD  - to refine
```

Comment:
was wondering about this ... do you think these pages should be moved out of the tls section?

Reply:
Probably that specific sectin should be removed from the overview, yes, as it's not a very common configuration use case, and it doesn't add much benefit. We can do it in a next PR to finalize the overview.